ospfd/ospf_main.c

```diff
@@ -134,6 +134,38 @@ FRR_DAEMON_INFO(ospfd, OSPF, .vty_port = OSPF_VTY_PORT,
 		.n_yang_modules = array_size(ospfd_yang_modules),
 );
 
+/** Max wait time for config to load before accepting hellos */
+#define OSPF_PRE_CONFIG_MAX_WAIT_SECONDS 600
+
+static void ospf_config_finish(struct event *t)
```

Comment:
Here  we are doing nothing upon this event expiry, Is it intended ?  what is the purpose of this event ?  
 I expect we should resume the hello packet forwarding upon this timer expiry .

Reply:
Exactly, we use the event state itself to suppress hellos so no further action is necessary.